src/components/Page/styles/Page.css.ts

```diff
@@ -1,8 +1,7 @@
-// @flow
 import baseStyles from '../../../styles/resets/baseStyles.css.js'
 import { breakpoint } from '../../../styles/mixins/breakpoints.css.js'
 import styled from '../../styled'
-import PageConfig from './Page.config.css.js'
+import PageConfig from './Page.config.css'
```

Comment:
@tinkertrain I deliberately kept `.css.js` as an extension for these imports to imply that the imported code was coming from a CSS-in-JS based file... rather than a standard `.css` file.

Unsure if that's necessary... Wanted to get your thoughts on this!! <3

Reply:
I actually do like a bit better to leave it at .css.js, the issue is that I get an error building because they are .css.ts files and it can find the .css.js (makes sense) but also you can't make the import with the .ts extension, so our options are:

a. Using TypeScript on css-in-js files: `import PageConfig from './Page.config.css'`
b. Don't use TypeScript on css-in-js files: `import PageConfig from './Page.config.css.js'`

My hunch is that we would want to have all in TS, so option **a**?

Reply:
Hmm!! Good point! Let's go with **a** 🙌